daycount.go

```diff
@@ -73,6 +73,17 @@ func yearFractionActualActualAFB(from, to date.Date) float64 {
 	nbFullYears := 0
 
 	remaining := to
+
+	// This function uses backward induction from the later date,
+	// 'from' and 'to' are swapped if 'to' falls before 'from'.
+	alpha := 1.0
```

Comment:
Should `alpha` be called `sign` or `directionality`?

Reply:
I asked myself the same question :)
It's generally called `signum` (at least in Java SDK)

Reply:
I like [signum](https://en.wikipedia.org/wiki/Sign_function)!